src/core/edit/edit.row.view.spec.js

```diff
@@ -0,0 +1,43 @@
+import {Command} from '../command';
+import {EditRowView} from './edit.row.view';
+import {modelFactory} from '../test/model.factory';
+import {CommandManager} from '../command/command.manager';
+
+describe('EditRowView', function () {
+	let commandManager = new CommandManager();
+	let model = modelFactory();
+	let table = {
+		view: {
+			focus: () => true
+		}
+	};
+	let editRowView = new EditRowView(model, table, commandManager);
+
+	describe('get commands', function () {
+		it('should return enter, commit, cancel, reset commands', () => {
+			let map = editRowView.commands;
+			expect(map.get('enter')).to.be.an.instanceOf(Command);
```

Comment:
there is no sence to check type of property, u should check execution results of these commands, canExecute & execute,